modules/apps/remote-app/remote-app-support-web/test/index.js

```diff
@@ -163,4 +163,56 @@ describe('remote-app-support-web', () => {
 			expect(receiveMessage).not.toHaveBeenCalled();
 		});
 	});
+
+	describe('"fetch" command', () => {
+		it('doesn\'t throw an error if "body" is undefined', async () => {
```

Comment:
Pro-Tip™: You could also go with "does not" to avoid the backslash escapes (that's what the other `it()` in this file do).

#NotBlockingFeedback